app/scripts/lib/setupSentry.js

```diff
@@ -64,36 +129,141 @@ export const SENTRY_BACKGROUND_STATE = {
   MetaMetricsController: {
     metaMetricsId: true,
     participateInMetaMetrics: true,
+    eventsBeforeMetricsOptIn: true,
   },
   NetworkController: {
-    networkId: true,
     networkStatus: true,
+    selectedNetworkClientId: false,
+    networkId: true,
     providerConfig: {
-      nickname: true,
-      ticker: true,
+      rpcUrl: false,
       type: true,
+      chainId: false,
+      ticker: true,
+      nickname: true,
+      rpcPrefs: false,
+      id: false,
+    },
+    networkConfigurations: false,
+    networksMetadata: {
+      EIPS: true,
+      status: true,
     },
   },
+  NftController: {
+    allNftContracts: false,
+    allNfts: false,
+    ignoredNfts: false,
+  },
   OnboardingController: {
     completedOnboarding: true,
     firstTimeFlowType: true,
     seedPhraseBackedUp: true,
   },
   PreferencesController: {
-    currentLocale: true,
     featureFlags: true,
-    forgottenPassword: true,
+    // double-check if contains PID?
```

Comment:
Probably best to exclude this. It might contain a secret (the only method we support today of using private ipfs servers is to embed the secret in the URL)

Reply:
Resolved in https://github.com/MetaMask/metamask-extension/pull/20595/commits/db9e2566e13811c8ddb6c590cc4370f8e74163f9